discovery-provider/src/queries/queries.py

```diff
@@ -1410,3 +1410,94 @@ def get_saves(save_type):
         save_results = helpers.query_result_to_list(query_results)
 
     return api_helpers.success_response(save_results)
+
+# Get the user saved collections & uploaded collections along with the collection user owners
```

Comment:
can you try to make it more clear in comment that this is one-off + a TODO or some note indicating that we should try to consolidate this logic in medium term